nextflow_schema.json

```diff
@@ -883,12 +884,14 @@
                     "type": "boolean",
                     "description": "Display help text.",
                     "fa_icon": "fas fa-question-circle",
+                    "default": false,
                     "hidden": true
                 },
                 "version": {
```

Comment:
🤯 didn't even know we had this parameter 😆

Reply:
that's fairly recent